sdk/rust/crates/dagger-sdk/src/core/mod.rs

```diff
@@ -1,4 +1,4 @@
-pub const DAGGER_ENGINE_VERSION: &'static str = "0.6.4";
+pub const DAGGER_ENGINE_VERSION: &'static str = "0.8.0";
```

Comment:
This change is relative to rust, I think you should open a different PR to include the version compatibility check in another language so we can merge PR fasters :)

Reply:
This PR work on PR https://github.com/dagger/dagger/pull/5565 which fixes this issue already. 🙇